src/panel/poi/blocks/Covid19.jsx

```diff
@@ -66,36 +70,61 @@ const getContent = ({ status, opening_hours, note, contribute_url }) => {
   return content;
 };
 
-const statusMessages = {
-  open: covidStrings.statusOpen,
-  open_as_usual: covidStrings.statusOpen,
-  maybe_open: covidStrings.statusMaybeOpen,
-  closed: covidStrings.statusClosed,
-  unknown: covidStrings.statusNoData,
-};
-
 /* eslint-disable */
-const Covid19 = ({ block }) => {
-  const statusMsg = statusMessages[block.status] || statusMessages['unknown'];
-
-  return <div className="poi_panel__info__section covid19">
-    <h4 className="poi_panel__sub_block__title">
-      <span className="covid19-tag">Covid-19</span>{statusMsg}
-    </h4>
-    {getContent(block)}
-    <div className="covid19-legalWarning">
-      <i className="icon-alert-triangle" />
+const LocalizedWarning = ({ countryCode }) => {
+  const warnings = {
+    'FR': (
       <div>
         <p>
-          Pendant toute la période de confinement, se déplacer vers ce lieu n'est autorisé
-          qu'en possession d'une attestation de déplacement dérogatoire.
+          {_('During the entire period of containment, movement to this location is only permitted with an overriding movement certificate.', 'covid19')}
         </p>
         <p>
-          Plus d'informations sur{' '}
-          <a rel="noopener noreferrer" href={covidConf.frInformationUrl}>interieur.gouv.fr</a>
+          {_('More information at', 'covid19')}
+          <a rel="noopener noreferrer" href={covidConf.frInformationUrl}> interieur.gouv.fr</a>
```

Comment:
The space shouldn't be part of the link. If you need one, you can use `{' '}` or put the link on the same line.